lib/basedriver/driver.js

```diff
@@ -140,6 +171,14 @@ class BaseDriver extends MobileJsonWireProtocol {
   // and ensuring that we execute commands one at a time. This method is called
   // by MJSONWP's express router.
   async executeCommand (cmd, ...args) {
+    // get start time for this command, and log in special cases
+    let startTime = Date.now();
```

Comment:
`process.hrtime([time])` might be a better, if what we care about is intervals. https://blog.tompawlak.org/measure-execution-time-nodejs-javascript